docs/source/releasenotes.rst

```diff
@@ -20,6 +20,16 @@ Release notes
 - Library **RPA.Browser.Selenium** (:pr:`502`): Automatically add URL scheme when
   navigating, such as `https` (default) or `http`. This functionality is controlled
   with the keyword `Set default URL scheme`.
+- Library **RPA.Cloud.AWS**:
```

Comment:
This moved now since we just released the rest of the notes.

Reply:
Merged master to resolve